app/src/test/java/fr/free/nrw/commons/nearby/NearbyAdapterFactoryTest.java

```diff
@@ -33,10 +33,13 @@
     private static final Place PLACE = new Place("name", Place.Label.AIRPORT,
             "desc", null, new LatLng(38.6270, -90.1994, 0), null);
     private static final Place UNKNOWN_PLACE = new Place("name", Place.Label.UNKNOWN,
+<<<<<<< HEAD
```

Comment:
Some conflicts still remain. :)

Reply:
They are from the branch actually, so I decided to make another commit to fix tests:)

Reply:
The annotations still remain though, better to get rid of those otherwise it won't compile. :)